tests/lib/acpi/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  acpi.basic:
+    tags:
+      - acpi
```

Comment:
maybe add acpica as a tag, this will then trigger tests when the sha for acpia in west manifest changes

Reply:
> maybe add acpica as a tag, this will then trigger tests when the sha for acpia in west manifest changes

Sure, I didn't realize we had such kind of automation (or that CI already identifies "acpica" as a valid tag.

Reply:
Done

Reply:
Is this documented feature?